spec/lib/remote_translations/microsoft/available_locales_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe RemoteTranslations::Microsoft::AvailableLocales do
+  describe ".available_locales" do
+    context "app locales that need to be mapping" do
+      it "includes locales with same format that I18n.available_locales" do
```

Comment:
I think there might be some typos here :thinking:.

```suggestion
      it "includes locales with the same format as I18n.available_locales" do
```